opentracing-metrics-prometheus-spring-autoconfigure/src/main/java/io/opentracing/contrib/metrics/prometheus/spring/autoconfigure/WebTracingConfigurationBeanPostProcessor.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2017 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.metrics.prometheus.spring.autoconfigure;
+
+import java.util.regex.Pattern;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+
+import io.opentracing.contrib.spring.web.autoconfig.WebTracingConfiguration;
+
+@Configuration
+@ConditionalOnClass(value = {WebTracingConfiguration.class})
+@ConditionalOnProperty(name="OPENTRACING_METRICS_EXPORTER_HTTP_PATH")
+public class WebTracingConfigurationBeanPostProcessor implements BeanPostProcessor {
+
+    @Value("${OPENTRACING_METRICS_EXPORTER_HTTP_PATH:false}")
```

Comment:
Spring question: if the bean is `ConditionalOnProperty`, it will be created only if that property exists, right? Does that mean that this line here will never use the default `false`, as this property will always be set?